contents/docs/user-guides/organizations-and-projects.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Organizations and projects
+sidebar: Docs
+showTitle: true
+---
+
+For a user experience that scales for all your needs, PostHog implements the concepts of organizations and projects.
+
+## Organizations
+
+An organization is the largest building block of PostHog's structure: it's made up of projects and has PostHog users as members.
+
+Most commonly a PostHog organization represents a real-world company. But this isn't a requirement and you are free to gather people any way you see fit.
+
+As a brand new PostHog user you'll always find yourself in a single organization upon account creation.
+If you've joined from an invite, that will be the organization you've been invited to.
+Otherwise you'll become the Owner of a new organization created just for you, based on the company name you provided when creating the account.
+
+To switch between organizations, to go the current organization's settings, or to create new projects, use the account dropdown on the right of the top bar.
+
+> **Note:** As a PostHog Cloud user, you can create, manage, and join organizations without limits forever. For self-hosted PostHog, however, multiple organizations belong to our premium team-oriented offering. To use this feature, [contact us for a self-hosted license](https://posthog.com/pricing).
+
+### Notifications
+
+By default every time a member joins an organization in an [email-enabled PostHog instance](/docs/self-host/configure/email) (including PostHog Cloud), all existing members of the organization get notified about this with an email. This increases security by making it explicit who gains access to your product data.
+
+If you'd rather not get such notifications however, you can simply disable setting "Email all current members when a new member joins" in section "Notification Preferences" of organization settings.
+
+### Adding new members
+
+Any organization member can create organization invites. Such an invite is valid **for 3 days** after creation and **only for the specified email**.
+
+If there's no account associated with that email, the invited person will have to create an account. Otherwise they'll be able to join with their existing account.
```

Comment:
```suggestion
If there's no account associated with that email, the invited person will have to create an account. Otherwise, they'll be able to join with their existing account.
```